data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/FlowEvent.avsc

```diff
@@ -14,8 +14,7 @@
         "net.corda.data.flow.event.Wakeup",
         "net.corda.data.flow.output.FlowStatus",
         "net.corda.data.flow.event.SessionEvent",
-        "net.corda.data.persistence.EntityResponse",
-        "net.corda.data.crypto.wire.ops.flow.FlowOpsResponse"
+        "net.corda.data.flow.event.external.ExternalEventResponse"
```

Comment:
Not related to your PR, but I've just noticed `FlowStatus` is in this list and it shouldn't be. Whoops!

Reply:
I dont think that's true right? The `FlowStatus` should never be received by the flow which is what `FlowEvent` is used for.

Reply:
I think we're saying the same thing? But the issue is it's already in there, so we're currently saying it can be received by the flow.

Reply:
oh I read it as "should be"... 🤦